recipe/meta.yaml

```diff
@@ -11,29 +11,35 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --use-deprecated=out-of-tree-build
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
-    - python >=3.6,<4.0
-    - poetry
+    - python
+    - poetry 1.4.0
```

Comment:
1.0 would be enough ? https://github.com/mariushelf/graphlib_backport/blob/73df515291003330c526d8700d0ea287c8ca04c0/pyproject.toml#L54